store/tikv/kv.go

```diff
@@ -253,8 +253,9 @@ func (s *tikvStore) BeginWithStartTS(startTS uint64) (kv.Transaction, error) {
 	return txn, nil
 }
 
-func (s *tikvStore) GetSnapshot(ver kv.Version) (kv.Snapshot, error) {
+func (s *tikvStore) GetSnapshot(ver kv.Version, priority int) (kv.Snapshot, error) {
```

Comment:
I think it is better to add an interface for kv.Snapshot such as `SetPriority()`.

Reply:
already change the interface